Sherchryst/Picscope
# Init project

## Description
- This PR merges the `init_project` branch into the `main` branch.
- It includes the initial setup of the Flutter project and the basic application structure.

## Changes
- Initialize Flutter project using `flutter create`
- Add project files to the Git repository
- Update `README.md` file with application description and features

## Benefits
- This PR sets the foundation for further development of the Picscope image search application.
- The changes made to the `README.md` file provide a clear understanding of the project for other developers.

## How to Test
- Clone the repository and checkout the `main` branch
- Verify that the basic application structure has been set up correctly
- Verify that the updated `README.md` file provides a clear understanding of the project

## Checklist
- [x] Changes have been thoroughly reviewed
